var/spack/repos/builtin/packages/antlr4-cpp-runtime/package.py

```diff
@@ -27,6 +27,8 @@ class Antlr4CppRuntime(CMakePackage):
     )
 
     depends_on("libuuid", type=["build", "link"], when="@:4.10.1")
+    depends_on("git", type=["build"])
+    depends_on("pkg-config", type=["build"])
```

Comment:
Does only `pkg-config` satisfy this dependency or could `pkgconf`?  If either could be used you can specify this as a dependency on the virtual package `pkgconfig` (see https://spack.readthedocs.io/en/latest/basic_usage.html#virtual-dependencies).

```
$ spack spack info --virtuals pkg-config
...
Virtual Packages: 
    pkg-config provides pkgconfig
...

$ spack providers pkgconfig
pkgconfig:
pkg-config  pkgconf
```